extensions/modulation.sigmf-ext.md

```diff
@@ -0,0 +1,196 @@
+
+|value|type|description|
+|----|----|-------|
+|`with_carrier`|(analog) with-carrier modulation|
+|`suppressed_carrier`|(analog) suppressed-carrier modulation|
+|`reduced_carrier`|(analog) reduced-carrier modulation|
+|`single_carrier`|(digital) single-carrier modulation|
+|`multi_carrier`|(digital) multi-carrier modulation|
+
+###### The `carrier_variant` pair
+
+The `symbol_variant` name can have the following values:
+
+|value|type|description|
+|----|----|-------|
+|`differential`|differential modulation|
+|`offset`|offset modulation (sometimes called 'staggered')|
+
+###### The `order` Pair
+
+The `order` name has an unsigned integer value that describes the modulation order, which typically refers to the number of symbols or states in a digital modulation (e.g., QAM64 has 64 symbols, QPSK has 4 symbols).
+
+###### The `duplexing` Pair
+
+The `duplexing` name can have the following values:
+
+|value|description|
+|----|-------|
+|`tdd`|time-division duplexing|
+|`fdd`|frequency-division duplexing|
+
+###### The `multiplexing` Pair
+
+The `multiplexing` name can have the following values:
+
+|value|description|
+|----|-------|
+|`tdm`|time-division multiplexing|
+|`fdm`|frequency-division multiplexing|
+|`cdm`|code-division multiplexing|
+|`ofdm`|orthogonal frequency-division multiplexing|
+|`sdm`|space-division multiplexing|
+|`pdm`|polarization-division multiplexing|
+
+###### The `multiple_access` Pair
+
+The `multiple_access` name can have the following values:
+
+|value|description|
+|----|-------|
+|`fdma`|frequency-division multiple access|
+|`ofdma`|orthogonal frequency-division multiple access|
+|`tdma`|time-division multiple access|
+|`cdma`|code-division multiple access|
+|`sdma`|space-division multiple access|
+|`pdma`|power-division multiple access|
+
+###### The `spreading` Pair
+
+The `spreading` name can have the following values:
+
+|value|description|
+|----|-------|
+|`fhss`|frequency-hopping spread spectrum|
+|`thss`|time-hopping spread spectrum|
+|`dsss`|direct-sequence spread spectrum|
+|`css`|chirp spread spectrum|
+
+###### The `bandwidth` Pair
+
+The `bandwidth` name has a float value that describes the channel bandwidth of the signal. Note that this is different from what is reported in the `core` namespace within an annotation, describing the occupied spectrum of a signal, which may or may not be comparable to the actual channel bandwidth of the communications system.
+
+###### The `system` Pair
+
+The `system` name is meant to provide a way to describe the higher-level system. For example, "802.11ac", "BlueTooth", or "LTE Release 12".
+
+## Examples
+
+Here is an example of a relatively simple modulation label, which describes a 10 kHz FM signal using time-division duplexing:
+
+    "modulation": {
+        "type": "analog",
+        "class": "fm",
+        "duplexing": "tdd",
+        "bandwidth": 10000.0
+    }
+
+Here is a more complex example that describes an LTE 5 MHz SC-OFDMA downlink:
```

Comment:
an OFDMA signal can have multiple `class`es in it, which is why I think OFDM should be placed in to the `class` object and possibly a new OFDM  object should be created which should be described by someone who has a need to describe detailed descriptions of OFDM signals.